test/functional/sqlsrv/sqlsrv_ae_insert_datetime.phpt

```diff
@@ -0,0 +1,112 @@
+--TEST--
+Test for inserting and retrieving encrypted datetime types data
+Bind params using sqlsrv_prepare without any sql_type specified
+--SKIPIF--
+<?php require('skipif_versions_old.inc'); ?>
+--FILE--
+<?php
+include 'MsCommon.inc';
+include 'AEData.inc';
+include 'MsSetup.inc';
+
+$dataTypes = array( "date", "datetime", "datetime2", "smalldatetime", "time", "datetimeoffset" );
+$conn = ae_connect();
+
+foreach ( $dataTypes as $dataType ) {
+    echo "\nTesting $dataType: \n";
+    
+    // create table
+    $tbname = GetTempTableName( "", false );
+    $colMetaArr = array( new columnMeta( $dataType, "c_det" ), new columnMeta( $dataType, "c_rand" ));
+    create_table( $conn, $tbname, $colMetaArr );
```

Comment:
I believe `c_rand` is supposed to be randomized?

Reply:
Yes my mistake, I'll change them